qutebrowser/browser/browsertab.py

```diff
@@ -755,7 +753,7 @@ def _on_load_finished(self, ok):
         self.load_finished.emit(ok)
         if not self.title():
             self.title_changed.emit(self.url().toDisplayString())
-        self._handle_auto_insert_mode(ok)
+        # self._handle_auto_insert_mode(ok)
```

Comment:
Remove this entirely instead of commenting it out :wink:

Reply:
Whoops x-x
I was sure I deleted it, I'm sooo out of shape. On it immediately

Reply:
Everything should be done, besides the testing part, I'm a bit confused about it.
For some reason I don't see my edits being shown here, might be a fluke on my side though.